templates/onboarding_survey.html

```diff
@@ -0,0 +1,123 @@
+{% extends "layout.html" %}
+{% block header %}
+<script>
+    document.addEventListener('DOMContentLoaded', function () {
+
+        // FIXME: hardercoded index
+        var racecheckboxes = document.querySelectorAll('input[type="checkbox"]');
```

Comment:
This should work.

Wouldn't it have been easier to have a `type=hidden` field that you just flip to not hidden as needed?

Reply:
I agree with that. I will make that change. I completely forgot about `type=hidden`